app/views/recipes/show.html.haml

```diff
@@ -19,6 +21,16 @@
   - if @recipe.forked_recipes_ids.any?
     = "Forks: #{@recipe.forked_recipes_ids.count}"
 
+%p= @recipe.description
+%p= @recipe.ingredients
+%p= @recipe.directions
+
+#rating
+  - if !@has_rated
```

Comment:
I would use the instance method we have access to (also, if you negate an `if` statement, please use `unless`):
```ruby
- unless recipe.rated_by?(current_user)
```